### PR TITLE
implement login exception

### PR DESCRIPTION
**Pull request by Sereg20**
Implement login exception logic

### CODE REVIEW

File: uimodule/webapp/view/ListReport.view.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
         <!-- DynamicPage Content -->
         <f:content>
-            <Table id="idProductsTable" 
+            <Table id="usersTableId" 
                 inset="false"
```
Comment (Artemische) [inline uimodule/webapp/view/ListReport.view.xml:58]: please check onFilterChange function and change table id in func too (71  line)

**Review by Artemische — CHANGES_REQUESTED** [review]
Are there any solutions without creating a new model

File: uimodule/webapp/view/ListReport.view.xml (hunk 1 above)
Comment (Artemische) [reply uimodule/webapp/view/ListReport.view.xml:58]: mb you need to sync dev changes

Comment (Sereg20) [reply uimodule/webapp/view/ListReport.view.xml:58]: Will change after merge

**Pull request merged**